Automattic/wp-calypso
# Add Jetpack Cloud Activity Log

#### Changes proposed in this Pull Request

* Support `/backups/activity` route
* Add new `ActivityLog` component to be shared between filtered backup views and the activity page

<img width="1392" alt="Screen Shot 2020-03-27 at 2 56 07 PM" src="https://user-images.githubusercontent.com/2810519/77803799-46b6fc00-703b-11ea-8245-c9f4c9fb94c3.png">

#### Testing instructions

1. Navigate to `/backups/activity` and verify after selecting a site you are taken to the activity log for that site
2. Reload the page and verify it takes you directly to the activity log for that site
3. Verify that the new activity log view works more or less the same as the filtered date view on the backups page.
